### DISCUSSION TITLE
[Feature Request] Open as webapp on iPhone

### DISCUSSION THREAD

**Discussion by Fishrider24** [daledavies/jump#81] [Feature Requests]
Is there a place I can add meta name="mobile-web-app-capable" content="yes" so it will open in its own window instead of with all my tabs in safari?  Thanks for your time!

**Comment by Fishrider24** [comment]
"meta name="mobile-web-app-capable" content="yes""

**Comment by daledavies** [comment]
Hi thanks for reaching out.  There isn't currently but I can look at adding a full range of mobile web app compatible markup as per this gist maybe... https://gist.github.com/hal0gen/5852bd9db240c477f20c

Probably add a config option to enable it. I'll consider this in my roadmap for the next release.

Converting this issue to a discussion too as it is a feature request.

**Comment by daledavies** [comment]
I've added this in commit https://github.com/daledavies/jump/commit/631f6f468f5d8f766503b87a5766ae4c8392b943, will be part of the next release :)

**Reply by Fishrider24** [reply]
You rock!
